app/views/notifier/review_orders.text.erb

```diff
@@ -1,3 +1,3 @@
-<%= text(".body", facility: @facility.to_s, account: @account.to_s) %>
+<%= text(".body", facility: @facility.to_s, accounts: @accounts.html_safe) %>
```

Comment:
What about the `smart: false` option? Would that avoid the conversion "smart" quotes and apostrophes?